src/CognitiveServices/CognitiveServices/help/Remove-AzCognitiveServicesCommitmentPlan.md

```diff
@@ -0,0 +1,123 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.CognitiveServices.dll-Help.xml
+Module Name: Az.CognitiveServices
+online version: https://learn.microsoft.com/powershell/module/az.cognitiveservices/remove-azcognitiveservicescommitmentplan
+schema: 2.0.0
+---
+
+# Remove-AzCognitiveServicesCommitmentPlan
+
+## SYNOPSIS
+Remove a Cognitive Services Commitment Plan
+
+## SYNTAX
+
+```
+Remove-AzCognitiveServicesCommitmentPlan [-ResourceGroupName] <String> [-Name] <String>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
```

Comment:
Add -PassThru for Remove-* cmdlets so that  remove cmdlets does not return an object by default. If this switch is specified, it returns true if successful.